app/kubemci/pkg/gcp/backendservice/backendservicesyncer.go

```diff
@@ -90,12 +92,19 @@ func (b *BackendServiceSyncer) DeleteBackendServices(ports []ingressbe.ServicePo
 func (b *BackendServiceSyncer) deleteBackendService(port ingressbe.ServicePort) error {
 	name := b.namer.BeServiceName(port.Port)
 	glog.V(2).Infof("Deleting backend service %s", name)
-	if err := b.bsp.DeleteGlobalBackendService(name); err != nil {
-		glog.V(2).Infof("Error in deleting backend service %s: %s", name, err)
-		return err
+	err := b.bsp.DeleteGlobalBackendService(name)
+	if err != nil {
+		if utils.IsHTTPErrorCode(err, http.StatusNotFound) {
+			fmt.Println("Backend service", name, "does not exist. Nothing to delete")
+			return nil
+		} else {
+			glog.V(2).Infof("Error in deleting backend service %s: %s", name, err)
```

Comment:
I see that at other places, we use fmt.Println. Lets do the same here?

Reply:
This one is different because it's Println'ed by the calling function. (though it loses the name of the service that failed), so changed.